hack/build.sh

```diff
@@ -258,16 +260,15 @@ basedir() {
 
 # Checks if a flag is present in the arguments.
 has_flag() {
-    filters="$@"
+    filters="$*"
```

Comment:
I would like to keep "$@" as this will honour quotes for flags, two (e.g. `--flag "my input"`). I'm not sure why shellchecks recommends $* here.